cmd/config/metrics-report.yml

```diff
@@ -300,6 +300,11 @@
   metricName: max-memory-sum-workers
   instant: true
 
+- query: sum( (node_memory_MemTotal_bytes - node_memory_MemAvailable_bytes) and on (instance) label_replace(kube_node_role{role="worker"}, "instance", "$1", "node", "(.+)") )
+  metricName: memory-sum-workers
+  instant: true
+  captureStart: true
```

Comment:
Adding this metric because it allows us to calculate the memory delta over the run, with start and end instant measurements.  This can help us detect memory leaks over the course of a run.